tmt/base.py

```diff
@@ -739,6 +739,8 @@ class Test(Core, tmt.export.Exportable['Test']):
     serialnumber: int = 0
 
     returncode: Optional[int] = None
+    real_starttime: Optional[str] = None
+    real_endtime: Optional[str] = None
```

Comment:
The `real_` prefix was introduced to prevent collision with the `duration` test key. Shall we call these just `starttime` and `endtime` if there's no name collision for them?

Reply:
Thanks!